Paymetheus.Tests.Decred/Sha256Tests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Paymetheus.Decred.Wallet;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Paymetheus.Tests.Decred
+{
+    public static class Sha256Tests
+    {
+        [Fact]
```

Comment:
This is a Theory not a Fact.  See http://xunit.github.io/docs/getting-started-desktop.html#write-first-theory.  Basically Facts test invariants in the code, while theories are for table driven tests.

This this is a test of just a single item in what could be a test vector, I would prefer to see this as a Theory.  The data can declared inline with InlineData attributes, or returned by other methods in this class using the MemberData attribute.  It would also be easier to encode the data as hexadecimal so that you can pass around the expected data as strings instead of byte arrays (which unfortunately can not be used with InlineData).